resources/views/widgets/collection.blade.php

```diff
@@ -9,6 +9,11 @@
             </a>
         </h2>
         @can('create', ['Statamic\Contracts\Entries\Entry', $collection])
+        <create-entry-button
+            button-class="btn-primary"
+            url="{{ $collection->createEntryUrl() }}"
+            :blueprints="{{ $blueprints->toJson() }}"
+            text="{{ $createLabel }}" />
```

Comment:
In blade, you can't have self closing components.

You gotta do `<create-entry-button ...></create-entry-button>`

When you fix that, then the `<a>` right after it will also show up. Was there a reason you left it?

Reply:
(I pushed a fix)